dandiapi/api/tests/test_version.py

```diff
@@ -156,10 +164,16 @@ def test_version_rest_update_large(api_client, user, draft_version):
         'list': ['a', 'b', 'c'],
         'very_large': 'words' * 10000,
     }
+    year = datetime.now().year
+    url = f'https://identifiers.org/{draft_version.dandiset.identifier}/draft'
     saved_metadata = {
         **new_metadata,
         'name': new_name,
         'identifier': f'DANDI:{draft_version.dandiset.identifier}',
+        'id': f'{draft_version.dandiset.identifier}/draft',
+        'version': 'draft',
+        'url': url,
+        'citation': f'{new_name} ({year}). Online: {url}',
     }
 
```

Comment:
Following my other comment, it might be good to have a specific test for the `citation` field, to test the possible conditions (I'd think mainly missing, empty, and populated).